### PR TITLE
Update sonic command doc to add CLIs relative to SONiC fips

### PR DESCRIPTION
**Pull request by xumia**
#### What I did
Update sonic command doc to add CLIs relative to SONiC fips.
See https://github.com/sonic-net/SONiC/blob/master/doc/fips/SONiC-OpenSSL-FIPS-140-3.md

### CODE REVIEW

File: doc/Command-Reference.md [hunk 1]
```diff
@@ -10397,6 +10397,38 @@ This command removes all unused images from the device, leaving only the current
   No image(s) to remove
   ```
 
+**sonic-install get-fips**
```
Comment (mdanish-kh) [inline doc/Command-Reference.md:10400]:
```suggestion
**sonic-installer get-fips**
```
Fix typo to match the previous command headings?

File: doc/Command-Reference.md [hunk 2]
```diff
@@ -10397,6 +10397,38 @@ This command removes all unused images from the device, leaving only the current
...
+  sonic-installer get-fips [<image_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-installer get-fips
+  FIPS is enabled
+  ```
+
+**sonic-install set-fips**
```
Comment (mdanish-kh) [inline doc/Command-Reference.md:10415]:
```suggestion
**sonic-installer set-fips**
```
Another one.

File: doc/Command-Reference.md (hunk 1 above)
Comment (xumia) [reply doc/Command-Reference.md]: @mdanish-kh , thanks for your review, fixed.

File: doc/Command-Reference.md (hunk 2 above)
Comment (xumia) [reply doc/Command-Reference.md]: > 



> Another one.

@mdanish-kh , thanks for your review, fixed.

**Comment by xumia** [comment]
@mdanish-kh, could you please help review again? Thanks.

File: doc/Command-Reference.md [hunk 3]
```diff
@@ -10397,6 +10397,38 @@ This command removes all unused images from the device, leaving only the current
...
 
+**sonic-installer get-fips**
+
+This command is used to get the fips status.
+
+- Usage:
+  ```
+  sonic-installer get-fips [<image_name>]
+  ```
+
```
Comment (mdanish-kh) [inline doc/Command-Reference.md:10408]:
```suggestion
If the image is not specified, the next boot image will be used.

```
Add a note.

**Review by mdanish-kh — APPROVED** [review]

File: doc/Command-Reference.md (hunk 3 above)
Comment (xumia) [reply doc/Command-Reference.md:10409]: @mdanish-kh , thanks for your suggestion.

**Comment by xumia** [comment]
@mdanish-kh , please approve again, thanks.

**Review by mdanish-kh — APPROVED** [review]
LGTM

**Review by liushilongbuaa — APPROVED** [review]

**Pull request merged**